libraries/lib-network-manager/CookiesList.cpp

```diff
@@ -0,0 +1,215 @@
+/*!********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file CookiesList.cpp
+ @brief Define HTTP cookies list class.
```

Comment:
Web APIs should not require cookies - they are unnecessarily complex. This reeks of bloat.